mk/targets.mk

```diff
@@ -46,6 +46,10 @@ pxf: HAL_BOARD = HAL_BOARD_LINUX
 pxf: TOOLCHAIN = BBONE
 pxf: all
 
+bebop: HAL_BOARD = HAL_BOARD_LINUX
+bebop: TOOLCHAIN = BBONE
```

Comment:
I agree with that, Especially because we will probably be switching our toolchain soon to the same as the BBONE one. So I will change that.